lottie/src/main/java/com/airbnb/lottie/model/layer/CompositionLayer.java

```diff
@@ -93,7 +94,11 @@ public CompositionLayer(LottieDrawable lottieDrawable, Layer layerModel, List<La
     for (int i = layers.size() - 1; i >= 0 ; i--) {
       boolean nonEmptyClip = true;
       if (!newClipRect.isEmpty()) {
-        nonEmptyClip = canvas.clipRect(newClipRect);
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.O_MR1) {
+            nonEmptyClip = canvas.clipOutRect(newClipRect);
```

Comment:
looking over documentation more this is the same as `clipRect(rect, Region.Op.DIFFERENCE)` which is not what we want. Fix coming.